packages/experimental/Switch/SPEC.md

```diff
@@ -0,0 +1,274 @@
+   */
+  thumbSize?: number;
+
+  /**
+   * Thumb radius
+   */
+  thumbRadius?: number;
+
+  /**
+   * Thumb margin
+   */
+  thumbMargin?: ViewStyle['margin'];
+
+  /**
+   * Color of border when Switch is focused on
+   */
+  focusStrokeColor?: ColorValue;
+
+  /**
+   * Width of border when Switch is focused on
+   */
+  focusBorderWidth?: number;
+
+  /**
+   * Border Radius of border when Switch is focused on
+   */
+  focusBorderRadius?: number;
+
+  /**
+   * Sets the position of the thumb
+   */
+  justifyContent?: ViewStyle['justifyContent'];
+
+  /**
+   * The width of the switch.
+   */
+  width?: ViewStyle['width'];
+
+  /**
+   * The minWidth of the switch
+   */
+  minWidth?: ViewStyle['minWidth'];
+
+  /**
+   * The minHeight of the switch
+   */
+  minHeight?: ViewStyle['minHeight'];
+
+  /**
+   * The flex direction of the root
+   */
+  flexDirection?: ViewStyle['flexDirection'];
+
+  /**
+   * The toggle container flex direction
+   */
+  toggleContainerFlexDirection?: ViewStyle['flexDirection'];
+
+  /**
+   * States that can be applied to a switch
+   */
+  hovered?: SwitchTokens;
+  focused?: SwitchTokens;
+  pressed?: SwitchTokens;
+  disabled?: SwitchTokens;
+  checked?: SwitchTokens;
+  toggleOn?: SwitchTokens;
+  toggleOff?: SwitchTokens;
+  before?: SwitchTokens;
+  after?: SwitchTokens;
+  above?: SwitchTokens;
+}
+```
+
+## Behaviors
+
+### States
+
+The following section describes the different states which the `Switch` can be in as a result of interaction.
+
+#### Enabled and Disabled states
+
+An enabled `Switch` communicates interaction by having styling that invites the user to click/tap on it to trigger a toggle.
+
+A disabled `Switch` is non-interactive, disallowing the user to click/tap on it to trigger a toggle.
+
+#### Hovered state
+
+A hovered `Switch` changes styling to communicate that the user has placed a cursor above it.
+
+#### Focused state
+
+A focused `Switch` changes styling to communicate that the user has placed keyboard focus on it. Specifically, a border is drawn around the control.
+
+#### Pressed state
+
+A pressed `Switch` changes styling to communicate that the user is currently pressing it.
+
+### Interaction
+
+#### Keyboard interaction
+
+The following is a set of keys that interact with the `Switch` component:
+
+| Key     | Description                                            |
+| ------- | ------------------------------------------------------ |
+| `Enter` | Executes the function passed into the `onChange` prop. |
+| `Space` | Executes the function passed into the `onChange` prop. |
+
+It is possible to override key behaviors by specifying `onKeyUp` or `onKeyDown`, depending on what `preferKeyDownForKeyEvents` from the `@fluentui-react-native/interactive-hooks` package indicates. Providing the callback this way will prevent the default behaviors noted above -- you will need to handle `Enter` and `Space` in the provided callback if the default behavior is desired.
```

Comment:
"noted above -- you will" - do you need two dashes here?

Reply:
I got that text from the Button's SPEC.md, I believe it is just an em dash. I think the em dash here is being used to stress a point about what will happen to the default behaviors.